### PR TITLE
Indonesia - Richard Yun

### PR DESCRIPTION
**Pull request by richardyun**

### CODE REVIEW

**Review by dpaschal-lf — CHANGES_REQUESTED** [review]
rejected.  Didn't have a screen shot of output

**Comment by richardyun** [comment]
indonesia

**Comment by richardyun** [comment]
Added screenshot.

File: flags.css [hunk 1]
```diff
@@ -1,5 +1,17 @@
 /* Indonesia */
-
+#indonesia {
+    height: 200px;
```
Comment (dpaschal-lf) [inline flags.css]: in the instructions, it does say to use pixels.  however vmin might be better for real usage.

Also, all your flags are going to be the same size, for the most part.  So you could add the size to the .flagContainer and get more bang for your buck.

File: flags.css [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+    width: 300px;
+    border: 1px solid black
```
Comment (dpaschal-lf) [inline flags.css]: be aware that adding a border to an element makes it bigger unless you set the box-sizing to border box

File: flags.css [hunk 3]
```diff
@@ -1,5 +1,17 @@
...
+}
+#indonesia > .topSegment {
+    height: 100px;
```
Comment (dpaschal-lf) [inline flags.css]: if your outer element sizes are sized correctly, it is definitely a better idea to have your internal elements as a %

File: flags.html [hunk 4]
```diff
@@ -4,80 +4,81 @@
         <link rel="stylesheet" href="flags.css">
     </head>
     <body>
-<div id="indonesia" class="flagContainer">
+        <div id="indonesia" class="flagContainer">
+            <div class="topSegment red"></div>
```
Comment (dpaschal-lf) [inline flags.html]: you have a class of red and white,. but you don't use that class for anything

File: flags.html [hunk 5]
```diff
@@ -4,80 +4,81 @@
...
+            <div class="bottomSegment white"></div>
+        </div>
+        <div id="columbia" class="flagContainer">
 
-</div>
-<div id="columbia" class="flagContainer">
+        </div>
```
Comment (dpaschal-lf) [inline flags.html]: watch out about having a prettifier in your code.  that can cause massive conflicts and unnecessary changes.  This is often from a plugin in your editor.

**Review by dpaschal-lf — CHANGES_REQUESTED** [review]
various problems

File: flags.html (hunk 5 above)
Comment (richardyun) [reply flags.html]: OK Dan. I looked in my VS code and the only extensions I have are live server, open in browser, and Power Mode... Don't think I have a prettifier.

**Comment by richardyun** [comment]
indonesia

File: flags.css [hunk 6]
```diff
@@ -1,17 +1,17 @@
+.flagContainer {
+    height: 66vmin;
+    width: 100vmin;
```
Comment (dpaschal-lf) [inline flags.css]: this is going to lead to some problems.  100% of the minimum viewport dimension+ 10% of the minimum viewport dimension (5% *2) is going to be 110% of the minimum viewport dimension.

**Review by dpaschal-lf — CHANGES_REQUESTED** [review]
110% of the viewport is no good.
